src/ansys/geometry/core/designer/body.py

```diff
@@ -1162,6 +1198,40 @@ def tessellate(  # noqa: D102
         else:
             return comp
 
+    @protect_grpc
+    def shell_body(self, offset: Real) -> bool:  # noqa: D102
+        check_type(offset, Real)
+
+        self._grpc_client.log.debug(f"Shelling body {self.id} to offset {offset}.")
+
+        result = self._commands_stub.Shell(
+            ShellRequest(
+                selection=self._grpc_id,
+                offset=offset,
+            )
+        )
+
+        return result.success
+
+    @protect_grpc
+    @reset_tessellation_cache
+    def remove_faces(self, selection: Union["Face", list["Face"]], offset: Real) -> bool:  # noqa: D102
+        selection: list[Face] = selection if isinstance(selection, list) else [selection]
+
+        check_type(offset, Real)
+        check_type_all_elements_in_iterable(selection, Face)
+
```

Comment:
Should we also verify that the faces belong to the body? or is it not necessary?